tests/contrib/operators/test_gcp_automl_operator.py

```diff
@@ -0,0 +1,383 @@
+
+INPUT_CONFIG = {"input": "value"}
+OUTPUT_CONFIG = {"output": "value"}
+PAYLOAD = {"test": "payload"}
+DATASET = {"dataset_id": "data"}
+MASK = {"field": "mask"}
+
+
+class TestAutoMLTrainModelOperator(unittest.TestCase):
+    @mock.patch("airflow.contrib.operators.gcp_automl_operator.AutoMLHook.create_model")
+    @mock.patch("airflow.contrib.operators.gcp_automl_operator.get_object_id")
+    def test_execute(self, mock_name, mock_service):
+        mock_xcom = mock.MagicMock()
+        mock_name.return_value = MODEL_ID
+        op = AutoMLTrainModelOperator(
+            model=MODEL,
+            location=GCP_LOCATION,
+            project_id=GCP_PROJECT_ID,
+            task_id=TASK_ID,
+        )
+        op.xcom_push = mock_xcom
+        op.execute(context=None)
+        mock_service.assert_called_with(
+            model=MODEL,
+            location=GCP_LOCATION,
+            project_id=GCP_PROJECT_ID,
+            retry=None,
+            timeout=None,
+            metadata=None,
+        )
+        mock_xcom.assert_called_with(None, key="model_id", value=MODEL_ID)
+
+
+class TestAutoMLBatchPredictOperator(unittest.TestCase):
+    @mock.patch(
+        "airflow.contrib.operators.gcp_automl_operator.AutoMLHook.batch_predict"
+    )
+    def test_execute(self, mock_service):
+        op = AutoMLBatchPredictOperator(
+            model_id=MODEL_ID,
+            location=GCP_LOCATION,
+            project_id=GCP_PROJECT_ID,
+            input_config=INPUT_CONFIG,
+            output_config=OUTPUT_CONFIG,
+            task_id=TASK_ID,
+        )
+        op.execute(context=None)
+        mock_service.assert_called_with(
+            input_config=INPUT_CONFIG,
+            location=GCP_LOCATION,
+            metadata=None,
+            model_id=MODEL_ID,
+            output_config=OUTPUT_CONFIG,
+            params={},
+            project_id=GCP_PROJECT_ID,
+            retry=None,
+            timeout=None,
+        )
+
+
+class TestAutoMLPredictOperator(unittest.TestCase):
+    @mock.patch("airflow.contrib.operators.gcp_automl_operator.AutoMLHook.predict")
+    def test_execute(self, mock_service):
+        op = AutoMLPredictOperator(
+            model_id=MODEL_ID,
+            location=GCP_LOCATION,
+            project_id=GCP_PROJECT_ID,
+            payload=PAYLOAD,
+            task_id=TASK_ID,
+        )
+        op.execute(context=None)
+        mock_service.assert_called_with(
+            location=GCP_LOCATION,
+            metadata=None,
+            model_id=MODEL_ID,
+            params={},
+            payload=PAYLOAD,
+            project_id=GCP_PROJECT_ID,
+            retry=None,
+            timeout=None,
+        )
+
+
+class TestAutoMLCreateImportOperator(unittest.TestCase):
+    @mock.patch(
+        "airflow.contrib.operators.gcp_automl_operator.AutoMLHook.create_dataset"
+    )
+    @mock.patch("airflow.contrib.operators.gcp_automl_operator.get_object_id")
+    def test_execute(self, mock_name, mock_service):
```

Comment:
```suggestion
    def test_execute(self, mock_get_object_id, mock_ create_dataset):
```
This will make it easier to read the code in the future.